tasks/src/app/app.component.ts

```diff
@@ -0,0 +1,10 @@
+import { Component } from '@angular/core';
+
+@Component({
+  selector: 'app-root',
+  templateUrl: './app.component.html',
+  styleUrls: ['./app.component.css']
+})
+export class AppComponent {
+  title = 'tasks';
```

Comment:
don't need anymore
```suggestion
```

Reply:
it is necessary to have a title, I think ❔

Reply:
in this case you don't use this variable anywhere

Reply:
it s used in app.component.spec.ts
Delete the sequence from there?

Reply:
delete the entire spec file

Reply:
ok, all good 😄